### DISCUSSION TITLE
Guilded Blackstone

### DISCUSSION THREAD

**Discussion by real-vgi** [DerFrZocker/Ore-Control#26] [Ideas]
Hello, I want to ask if there is a way to stop generating Guided Blackstone. This block drops alot of gold when broke and I think it should be on the list of blocks u can stop generating. 

I run an item based economy with Gold Ingots as currency, and I prefer to use ore-control to disable the spawn of gold ore. 
But Guilded Blackstone has the same effect as a Nether Gold Ore block.

**Comment by DerFrZocker** [comment]
Gilded blackstone is generated as part of the bastion remnant structure and not as a separated feature, as the other ores.
Which is why Ore-Control is not suitable for this use case. However, since it is relative easy to remove gilded blackstone in 1.20.2 and up from a structure, I made a simple plugin which you can use for this and find [here](https://www.spigotmc.org/resources/114762/).

**Reply by real-vgi** [reply]
Wow, thank you so much.

Writing an entire plugin for this issue says you are a very talented and dedicated developer.

Thank you again for your work.
